### PR TITLE
ui-columncontrols-clarification-7730

### PR DESCRIPTION
**Pull request by Santoshziffity**
## Purpose of this pull request

This pull request adds some example to this topic 
Topic name : ui-columncontrols-clarification-7730

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/ui_comp_guide/components/ui-columnscontrols.html

## Links issues : 
#7730 

whatsnew
Updated the example in the [ColumnsControl component](https://devdocs.magento.com/guides/v2.4/ui_comp_guide/components/ui-columnscontrols.html) topic.

### CODE REVIEW

File: src/guides/v2.3/ui_comp_guide/components/ui-columnscontrols.md [hunk 1]
```diff
@@ -25,22 +25,51 @@ Extends [`uiCollection`]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uicol
...
-         <columnsControls name="columns_controls">
-            <settings>
-                <minVisible>1</minVisible>
-                <maxVisible>3</maxVisible>
-            </settings>
-         </columnsControls>
-    </listingToolbar>
-    ...
-</listing>
+### You can simply change "minVisible" and "maxVisible" values in the JS file.
```
Comment (BarnyShergold) [inline src/guides/v2.3/ui_comp_guide/components/ui-columnscontrols.md:28]: Headings should be short and non-conversational
```suggestion
### Change "minVisible" and "maxVisible" values in the JS file.
```

File: src/guides/v2.3/ui_comp_guide/components/ui-columnscontrols.md [hunk 2]
```diff
@@ -25,22 +25,51 @@ Extends [`uiCollection`]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uicol
...
+
+For this, you have to override the vendor JS file in your custom module.
+
+vendor file js path :
+/vendor/magento/module-ui/view/base/web/js/grid/controls/columns.js
+
+#### For override JS file follow step Below:
```
Comment (BarnyShergold) [inline src/guides/v2.3/ui_comp_guide/components/ui-columnscontrols.md:35]:
```suggestion
#### To override the JS file follow the step below:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few small changes

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**